api/controllers/arithmeticController.js

```diff
@@ -11,7 +11,7 @@ exports.calculate = function(req, res) {
   });
 
   var operations = {
-    'add':      function(a,b) { return +a + +b },
+    'add':      function(a,b) { return a + b },
```

Comment:
Add the + sign back to this line's return vars. JS will concatenate them as strings if they are not present.

Reply:
Thanks for adding. Javascript string is new to me!

Reply:
That's ok!! ✨